documentation/Functions.md

```diff
@@ -266,9 +266,11 @@ Repository will be cloned into a newly created subdirectory devonfw-tutorials.
 
 
 ### runServerJava <a name="runServerJava"></a>
+This function will run your java application on server.Parameter and Assertion information you need to use properly, which is mentioned below.
```

Comment:
This function will run your Java application. Parameter and assertion information you need to use properly, which is mentioned below. In Katacoda the command will be executed in a new terminal.